live-editing/configs/DateTimeEditorConfigGenerator.ts

```diff
@@ -0,0 +1,28 @@
+import { IgxDateTimeEditorDirective, IgxInputGroupModule } from "igniteui-angular";
```

Comment:
Remove IgxDateTimeEditorDirective from DateTimeEditorConfigGenerator and replace it with IgxDateTimeEditorModule
```suggestion
import { IgxDateTimeEditorDirective, IgxInputGroupModule } from "igniteui-angular";
```